WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -313,7 +313,7 @@ def my_competitions
     if current_user.person
```

Comment:
It's not your fault that `my_competitions` doesn't have a test, but something this complex really needs one!